src/NetAppFiles/NetAppFiles/Subvolume/RemoveNetAppFilesSubvolume.cs

```diff
@@ -0,0 +1,162 @@
+
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true,
+            ParameterSetName = ResourceIdParameterSet,
+            HelpMessage = "The resource id of the ANF Subvolume")]
+        [ValidateNotNullOrEmpty]
+        public string ResourceId { get; set; }
+
+        [Parameter(
+            ParameterSetName = ParentObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The volume object containing the subvolume to remove")]
+        [ValidateNotNullOrEmpty]
+        public PSNetAppFilesVolume VolumeObject { get; set; }
+
+        [Parameter(
+            ParameterSetName = ObjectParameterSet,
+            Mandatory = true,
+            ValueFromPipeline = true,
+            HelpMessage = "The subvolume object to remove")]
+        [ValidateNotNullOrEmpty]
+        public PSNetAppFilesBackup InputObject { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Return whether the specified backup was successfully removed")]
+        public SwitchParameter PassThru { get; set; }
+
+        public override void ExecuteCmdlet()
+        {
+            bool success = false;
+            if (ParameterSetName == ResourceIdParameterSet)
+            {
+                var resourceIdentifier = new ResourceIdentifier(this.ResourceId);
+                ResourceGroupName = resourceIdentifier.ResourceGroupName;
+                var parentResources = resourceIdentifier.ParentResource.Split('/');
+                AccountName = parentResources[1];
+                PoolName = parentResources[3];
+                VolumeName = parentResources[5];
+                Name = resourceIdentifier.ResourceName;
+            }
+            else if (ParameterSetName == ObjectParameterSet)
+            {
+                ResourceGroupName = InputObject.ResourceGroupName;
+                var NameParts = InputObject.Name.Split('/');
+                AccountName = NameParts[0];
+                PoolName = NameParts[1];
+                VolumeName = NameParts[2];
+                Name = NameParts[3];
+            }
+            else if (ParameterSetName == ParentObjectParameterSet)
+            {
+                ResourceGroupName = VolumeObject.ResourceGroupName;
+                var NameParts = VolumeObject.Name.Split('/');
+                AccountName = NameParts[0];
+                PoolName = NameParts[1];
+                VolumeName = NameParts[2];
+            }
+
+            if (ShouldProcess(Name, string.Format(PowerShell.Cmdlets.NetAppFiles.Properties.Resources.RemoveResourceMessage, Name)))
+            {
+                AzureNetAppFilesManagementClient.Subvolumes.Delete( ResourceGroupName, accountName: AccountName, poolName: PoolName, volumeName: VolumeName, subvolumeName: Name);                
+            }
+
+            if (PassThru)
+            {
+                WriteObject(success);
```

Comment:
How can we let success be true? In common practice, if this command runs successfully, we will return true.